tasks/monitoring.yml

```diff
@@ -0,0 +1,58 @@
+- name: Centralize VPN Logs
+  watch_logs:
+    name: openvpn
+    file_paths:
+      - "{{ openvpn_log }}"
+
+- name: Create directory for nagios plugins
+  file:
+    state: directory
+    dest: "{{ openvpn_etcdir }}/nagios-plugins"
```

Comment:
could have used the already defined variable "openvpn_nagios_plugin_dir" for easy understanding ??